src/js/_enqueues/wp/wp-image-mime-fallback-loader.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @output wp-includes/js/wp-image-mime-fallback-loader.js
+ */
+
+( function( document, image_src, settings ) {
```

Comment:
Is `image_src` a good name for this value? Not saying it's not, but it is not the full value passed to the `src` attribute, so maybe there is a more suitable one? Any idea @eugene-manuilov? Do you remember how you originally named this variable in the plugin before shortening it to `p`?

Reply:
`image_src` was removed from the function variable and use as normal variable. Please check and let me know if we need any changes